packages/xod-client-electron/src/shared/events.js

```diff
@@ -34,8 +34,11 @@ export const APP_UPDATE_DOWNLOAD_REQUEST = 'EVENT_APP_UPDATE_DOWNLOAD_REQUEST';
 export const APP_UPDATE_DOWNLOAD_STARTED = 'EVENT_APP_UPDATE_DOWNLOAD_STARTED';
 
 export const START_DEBUG_SESSION = 'START_DEBUG_SESSION';
-export const DEBUG_SESSION = 'DEBUG_SESSION';
+export const SEND_TO_DEBUG_SERIAL = 'SEND_TO_DEBUG_SERIAL';
+export const MESSAGES_FROM_SERIAL_SESSION = 'MESSAGES_FROM_SERIAL_SESSION';
```

Comment:
The proper form of AT name is `<NOUN>_<VERBINFINITIVE>` or consistent to surroundings. This seems like neither of cases.

Should it better be:

- `DEBUG_SERIAL_SEND`
- `SERIAL_SESSION_MESSAGE_RECEIVE` (or `SERIAL_SESSION_RECEIVE`)